src/views/ranking/RankingView.vue

```diff
@@ -227,6 +280,11 @@ const restRankUsers = RANKING_LIST.filter(user => user.rankingNow > 3);
 
 const showModal = ref(true);
 
+// 추구미 유형 이름 매핑
+function getChoogoomiType(typeCode) {
+  return CHOOGOOMI_MAP[typeCode]?.[0]?.choogoomiType || '';
```

Comment:
추구미를 선택해야 서비스 이용이 가능하니까 타입이 없는 경우는 없을 거 같습니다!! 지금 수정하실 필요는 없고 실제로 API연결할때 이부분을 고려하면 좋을 것 같습니다!!
```suggestion
  return CHOOGOOMI_MAP[typeCode].[0].choogoomiType;
```